aiplatform/snippets/embeddings.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START aiplatform_text_embeddings]
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	aiplatform "cloud.google.com/go/aiplatform/apiv1"
+	aiplatformpb "cloud.google.com/go/aiplatform/apiv1/aiplatformpb"
```

Comment:
nit: There's no need to name this, since it will be called aiplatformpb anyway.
```suggestion
	"cloud.google.com/go/aiplatform/apiv1/aiplatformpb"
```

Reply:
Done